compiler/src/main/scala/dx/translator/wdl/RuntimeTranslator.scala

```diff
@@ -77,7 +78,8 @@ case class RuntimeTranslator(wdlVersion: WdlVersion,
                              metaSection: Option[TAT.MetaSection],
                              defaultAttrs: Map[String, Value],
                              evaluator: Eval,
-                             dxApi: DxApi = DxApi.get) {
+                             dxApi: DxApi = DxApi.get,
+                             logger: Logger = Logger.get) {
```

Comment:
What's the purpose of adding `logger`?

Reply:
Probably used it for testing and forgot to remove.